lib/awful/widget/tasklist.lua

```diff
@@ -430,6 +434,8 @@ end
 --   update. See `awful.widget.common.list_update`.
 -- @tparam[opt] table args.layout Container widget for tag widgets. Default
 --   is `wibox.layout.flex.horizontal`.
+-- @tparam[opt=awful.tasklist.source.all_clients] function args.source The
```

Comment:
mb `source_function`, to be inline with `update_function`?

Reply:
filter isn't called "filter_function". So I am not sure about this...

Reply:
So... What's your opinion? Is my answer ok or do you really want this change? I am waiting for this to be merged to propose the pull request that uses it.